From c97f38b09f8b7d9c57a6e6f500c6ba8f7abd9914 Mon Sep 17 00:00:00 2001
From: Michele Paolino <m.paolino@virtualopensystems.com>
Date: Fri, 19 May 2017 14:50:55 +0200
Subject: [PATCH] Boot Normal World in EL2

This patch configures ATF (SPSR register) to boot BL33 in EL2.

Signed-off-by: Michele Paolino <m.paolino@virtualopensystems.com>
---
 plat/renesas/rcar/platform.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/renesas/rcar/platform.mk b/plat/renesas/rcar/platform.mk
index ef7bf0f..06bc592 100644
--- a/plat/renesas/rcar/platform.mk
+++ b/plat/renesas/rcar/platform.mk
@@ -186,7 +186,7 @@ $(eval $(call add_define,RCAR_DRAM_SPLIT))
 
 # Process RCAR_BL33_EXECUTION_EL flag
 ifndef RCAR_BL33_EXECUTION_EL
-RCAR_BL33_EXECUTION_EL := 0
+RCAR_BL33_EXECUTION_EL := 1
 endif
 $(eval $(call add_define,RCAR_BL33_EXECUTION_EL))
 
-- 
2.7.4

